doc/crud.md

```diff
@@ -268,6 +362,11 @@ This is the simplest case as we don't need to worry about previous revisions.
     CDTDocumentRevision *saved = [datastore createDocumentFromRevision:rev
                                                                  error:&error];
     ```
+    ```swift
+    let rev = CDTDocumentRevision("doc1")
+    rev.body = [....]
```

Comment:
I've started to use block comments in the eliding situation because it's clearer, e.g., `/* ... */`. As a side effect, it tends not to get some weird highlighting.